spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+RSpec.describe NotificationMailer, type: :mailer do
+  describe 'release_email' do
+    let(:user) { mock_model User, first_name: 'Ignas', last_name: 'Panero Armoska', email: 'ignaspan@gmail.com' }
```

Comment:
We're actually using [FactoryBot](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md) to create mock objects. Something like `let(:user) { create(:user) }` should work fine.

Reply:
(And similarly below as well.)